e2etests/pkg/frr/bgp.go

```diff
@@ -110,29 +110,15 @@ type FRRRoute struct {
 	} `json:"nexthops"`
 }
 
+// BFDPeer represents a BFD peer.
 type BFDPeer struct {
```

Comment:
why zap the other fields? some linter complained?

Reply:
Don't remember, probably we weren't using it and I found it confusing.